### ISSUE TITLE
Can't get Spoolman running 

### ISSUE THREAD

**Issue by SinglerGold2** [Donkie/Spoolman#493]
Trying to install Spoolman on a BTT 7 with CM4

while  the install procedure runs fine, I can't get it running. 
I reinstalled it, setup the OS fresh etc... nothing helps

This is what is happening when I run ./scripts/start.sh:

![image](https://github.com/user-attachments/assets/22d31653-5d9c-4f58-88d5-5e7af571042c)

My knowledge about Linux is... limited, so any help is appreciated

**Spoolman Host (please complete the following information):**
 - Installation type: Stand Alone
 - Installed version: 0.20.0 (fresh install, not an update)
 - OS and Distro:  Mainsail-OS (Debian GNU/Linux 11 (bullseye)) on a BTT Pad7 with Raspberry CM4 4gb
 - Database type: the one that comes with Spoolman

**Desktop (please complete the following information):**
 - OS: Win11
 - Browser: Edge

https://github.com/user-attachments/assets/a33efb68-bdf0-47f8-b4d1-9e7be2ef79fc

**Comment by SpaceRichard** [comment]
I have the same issue, updates from 0.19.0 to 0.20.0 and it can't not start now

**Comment by SinglerGold2** [comment]
I - somehow - also installed 0.19.3 and this version works....

**Comment by Uro1** [comment]
The `.env.example` file in the repository which the as-installed `.env` configuration file is copied/created from has Windows CRLF line terminations instead of Unix/Linux LF line terminations, this creates the reported errors.

You can either install dos2unix to correct the files using that tool or run both `.env` and `.env-example` through `sed` and use the `file` command afterwards to verify the additional \r line terminators from CRLF have been removed:

```bash
sed -i 's/\r$//' .env .env.example
file .env && file .env.example
```

Doing this change will strip off the additional `\r` from the CRLF Windows/DOS line endings which is creating this issue and returning them to have Linux LF line endings.

After correcting the `.env` file line endings simply restart the Spoolman service with `sudo systemctl restart Spoolman` and the service will start normally and in a functional state.

I have submitted a [pull request]( https://github.com/Donkie/Spoolman/pull/497) that should fix this issue at the root cause going forwards for subsequent released, but for anyone with a broken install of 0.20.0 and unable to revert to a previos version the above instructions should get you going.


![terminal](https://github.com/user-attachments/assets/c9a29d57-2ff0-4e9a-b761-ec26f6f2b6f7)

**Comment by Donkie** [comment]
I have patched the 0.20 release zip with a fixed .env.example now so this should be resolved. If you haven't yet gotten Spoolman started, remove the spoolman folder and retry the installation steps again.
